fuel-tx/src/transaction/policies.rs

```diff
@@ -0,0 +1,329 @@
+use fuel_types::{
+    canonical::{
+        Deserialize,
+        Error,
+        Input,
+        Output,
+        Serialize,
+    },
+    BlockHeight,
+    Word,
+};
+
+#[cfg(feature = "random")]
+use rand::{
+    distributions::{
+        Distribution,
+        Standard,
+    },
+    Rng,
+};
+
+bitflags::bitflags! {
+    /// See https://github.com/FuelLabs/fuel-specs/blob/master/src/tx-format/policy.md#policy
+    #[derive(Clone, Copy, Default, Debug, PartialEq, Eq, Hash)]
+    #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+    pub struct PoliciesBits: u32 {
```

Comment:
Is there a particular reason to use `u32`?

Reply:
https://github.com/FuelLabs/fuel-specs/pull/514#discussion_r1366359260